### ISSUE TITLE
Value (i.e. "de") is stored in database as value and label

### ISSUE THREAD

**Issue by wompip** [terminal42/contao-countryselect#5]
Hi there,
cannot get the label (i.e. "Deutschland") in the email or the export/ csv-file, no matter which export configuration i choose. Because instead of the label "Deutschland" there is a second time "de" stored in the database.

**Comment by aschempp** [comment]
This extension provides a form field, and it has nothing to do with csv export, emails or storing in the database. The field behaves exactly the same as any Contao field, the value is stored and not the label. You should probably refer to the extensions you use to save the value.

**Comment by wompip** [comment]
Ok, sorry if I missunderstood something. I thought that the countryselect-field should behave in the same way that a usual select field does. That's why I made a test with countryselect (field 'land") and a usual select field ("land2"). In "land2" I called the value "wert2" an the label "Bezeichnung2". Both are in the database (see Screenshot). The line above ("land") comes from the countryselect-field (value "et" and label "et" instead of "Äthiopien"). Sorry - I just want to be sure if this is the correct behavior.
![screenshot-phpmyadmin](https://cloud.githubusercontent.com/assets/15787622/22332810/1b9c24ca-e3d3-11e6-85b2-eb4c3eb40478.jpg)
